src/build_workflow/build_target.py

```diff
@@ -54,8 +58,11 @@ def component_version(self):
 
     @property
     def compatible_component_versions(self):
-        # BUG: the 4th digit is dictated by the component, it's not .0, this will break for 1.1.0.1
```

Comment:
Can you restore this comment as this is still the case.
Thanks.

Reply:
This is no longer the case.

Reply:
Got it.